checklists/avd_checklist.en.json

```diff
@@ -2,478 +2,253 @@
-            "guid": "90083845-c587-4cb3-a1ec-16a1d076ef9f",
-            "severity": "High",
-            "link": "https://docs.microsoft.com/azure/virtual-desktop/app-attach-file-share"
-        },
-        {
-            "category": "Compute",
-            "subcategory": "MSIX & AppAttach",
-            "text": "Review performance considerations for MSIX",
-            "description": "In the referenced article, we reported few but important performance considerations for MSIX usage in AVD context, be sure to carefully review.",
-            "guid": "241addce-5793-477b-adb3-751ab2ac1fad",
-            "severity": "High",
-            "link": "https://docs.microsoft.com/azure/virtual-desktop/app-attach-file-share"
-        },
-        {
-            "category": "Compute",
-            "subcategory": "MSIX & AppAttach",
-            "text": "Check proper session host permissions for MSIX share",
-            "description": "MSIX app attach requires read-only permissions to access the file share. If you're storing your MSIX applications in Azure Files, then for your session hosts, you'll need to assign all session host VMs both storage account role-based access control (RBAC) and file share New Technology File System (NTFS) permissions on the share.",
-            "guid": "66e15d4d-5a2a-4db2-a3e2-326bf225ca41",
-            "severity": "Medium",
-            "link": "https://docs.microsoft.com/azure/virtual-desktop/app-attach-file-share"
-        },
-        {
-            "category": "Compute",
-            "subcategory": "MSIX & AppAttach",
-            "text": "MSIX packages for 3rd-party applications",
-            "description": "3rd-party software vendor must provide a MSIX package, it is not recommended for customer to attempt  the conversion procedure without proper support from the application owner.",
-            "guid": "bd362caa-ab79-4b19-adab-81932c9fc9d1",
-            "severity": "Medium",
-            "link": "https://docs.microsoft.com/azure/virtual-desktop/app-attach-faq"
+            "link": "https://docs.microsoft.com/azure/virtual-desktop/configure-host-pool-personal-desktop-assignment-type"
         },
         {
-            "category": "Compute",
-            "subcategory": "MSIX & AppAttach",
-            "text": "Disable auto-update for MSIX packages",
-            "description": "MSIX app attach doesn't support auto-update for MSIX applications, then should be disabled.",
-            "guid": "bb88037f-5e6b-4fbb-aed5-03547cc447e8",
+            "category": "Foundation",
+            "subcategory": "Capacity Planning",
+            "text": "For Pooled Host Pool type, select the best load balancing method",
+            "description": "Check which one to use and available options, be aware that if autoscaling will be used, it sets it to breadth-first.",
```

Comment:
Replace "be aware that if autoscaling will be used, it sets it to breadth-first." with "autoscale ignores existing load-balancing algorithms."